server.js

```diff
@@ -37,13 +38,41 @@ app.get('/api/sessions/:film', (req, res) => {
         return res.status(500).json({ message: 'Запрос не выполнен' });
       }
 
-      const filmName = data[0].name || '';
-
-      res.json({
-        result: data[0].sessions,
-        filmName
-      });
+      if (data.length === 0) {
+        res.status(404).json({ message: 'Такого фильма не существует' });
+      } else {
+        res.json({
+          result: data[0].sessions,
+          filmName: data[0].name
+        });
+      }
     });
 });
 
+app.post('/api/ticket', (req, res) => {
+  const ticket = new Ticket({
```

Comment:
You should add data validation here. What if sessionId is wrong? What if name is empty? What if I pass 'cat' in number of seats? What if number of seats is greater then number of seats actually available for session?